.gitignore

```diff
@@ -21,4 +21,4 @@ debug
 /mac_install.log
 /win_install.log
 /linux_install.log
-/src/default_excludes.mk
+/src/default_excludes.mk
```

Comment:
This should not be here please remove it as it did not change other than removing the last \n which is wrong anyway.